app/src/main/java/com/spascoding/taskycourse/core/DataError.kt

```diff
@@ -0,0 +1,15 @@
+package com.spascoding.taskycourse.core
```

Comment:
The core package should also be separated into presentation/domain/data with this type of architecture, since every type of shared class can also be clearly attributed to one layer. This one here to core->domain for example